src/components/DataViewerCell.vue

```diff
@@ -0,0 +1,32 @@
+<template functional>
+  <td
+    :class="{'data-viewer-cell': true, 'data-viewer-cell--active': props.isSelected }"
+  >{{ props.value }}</td>
+</template>
+<script lang="ts">
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+
+/**
+ * @name DataViewerCell
+ * @description A cell display in a DataViewer
+ * @param {boolean} isSelected - Wether the cell is selected or is in a selected column
+ * @param {string} value - The cell's value
+ */
+@Component({
+  name: 'data-viewer-cell',
```

Comment:
I think you should make this component functional as:
- it has no state
- the number of cells can be quite high, and the memory I'm of so many components is not to be neglected
- there is no real interest seeing this component in vue dev tools

Reply:
It is, the declaration is done on the template